### DISCUSSION TITLE
Why does SuppressWarnings annotation not work for CommentSize?

### DISCUSSION THREAD

**Discussion by linusjf** [pmd/pmd#2995] [Q&A]
https://raw.githubusercontent.com/Fernal73/LearnJava/master/Cryptography/cryptic/DHKeyAgreement.java

Why does `@SuppressWarnings("PMD.CommentSize")` not work for the above java class in function main?

**Comment by oowekyala** [comment]
This is now a bug report: #2996

**Reply by linusjf** [reply]
I did refer this in the bug report. However, it did not create a back link here.
